### DISCUSSION TITLE
Add Crowd play and crowd choice status into the extended detail

### DISCUSSION THREAD

**Discussion by Lordubuntu94** [ChristopherKlay/StadiaEnhanced#176] [Ideas & Requests]
title says it all :D

Just a simple supported, not supported

**Comment by ChristopherKlay** [comment]
The data for the extended details section isn't managed by me, but OriginalPenguin.

I'll forward the request (:

**Comment by OriginalPenguin94** [comment]
> title says it all :D
> 
> Just a simple supported, not supported

Will work on adding that information to the database when I get chance. 🙂
